index.html

```diff
@@ -0,0 +1,245 @@
+            <li>Terminal</li>
+        </ul>
+        </div>
+  </section>
+
+  <hr class="separation-line" />    
+  
+  <div class="social-media-icons">
+    <ul>
+      <li>
+        <a href="#"><i class="fab fa-github"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-linkedin-in"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-instagram"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-twitter"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-facebook"></i></a>
+      </li>
+    </ul>
+  </div>
+  
+<section class="my-works-section">
+    <div class="my-works-titles">
+      <i class="fa fa-arrow-down my-work-icon"></i>
+      <h3 class="my-works-section-title">Work</h3>
+    </div>
+
+    <h2 class="my-recent-work">MY RECENT WORK</h2>
+    <div class="my-works-section-separator">
+        <hr class="separation-line" />
+    </div>
+
+    <div class="my-projects">
+      <div class="my-project-images-container">
+        <img
+          src="../images/background.jpg"
+          alt="project image"
+          class="my-project-image"
+        />
+      </div>
+
+      <div class="my-project-info-1">
+        <h2 class="my-project-name">Multi-Post Stories</h2>
+        <p class="my-project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+
+      <div class="project-technologies">
+        <ul>
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+      <div>
+        <button type="button" class="my-project-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
+          alt="project image"
+          class="my-project-image"
+        />
+      </div>
+
+      <div class="my-project-info-2">
+        <h2 class="my-project-name">Multi-Post Stories</h2>
+        <p class="my-project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+
+      <div class="project-technologies">
+        <ul>
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+      <div>
+        <button type="button" class="my-project-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
```

Comment:
- [ ] Please kindly fix this image `src`, as you can notice in the screenshot below the image is broken, the `src` is src="./images/background.jpg" with one point to track the path for your image

![Capture](https://user-images.githubusercontent.com/80895497/148665633-79dad873-f1e6-4225-941c-e2138a6b9ff2.PNG)